docs/activity.md

```diff
@@ -0,0 +1,241 @@
+
+And then drag and drop the hex file to the Python Editor. The editor might
+warn you that this action will replace the project, you can click "Replace" to
+accept it.
+
+### So... What am I looking at?
+
+The loaded hex file contains two Python files, to see them let's go to "Project"
+section on the left sidebar:
+
+![Python Editor Projects](img/editor-project.png)
+
+In this view you can switch between the `main.py` file and the `bird.py`.
+
+- The `bird.py` module contains some functionality for the bird to be able to
+react to the world.
+- The `main.py` file is the main code we will be working on.
+
+Both files have some comments that should help clarify what some of the
+code is doing, but there is no need to familiarise yourself with these yet,
+as we'll work through the files as part of this activity.
+
+The sidebar also have other useful sections:
+- Reference: A place to explore the features of your micro:bit with Python
+- API: The micro:bit MicroPython API developer documentation
+
+<img width="45%" alt="microbit-universal-hex logo" src="/microbit-bird-activity/img/editor-reference.png">
+<img width="45%" alt="microbit-universal-hex logo" src="/microbit-bird-activity/img/editor-api.png">
+
+
+## Reacting to events
+
+You'll see at the top of `main.py` a function called `hello()` with a decorator
+`@bird.react("hello")` on top:
+
+```python
+@bird.react("hello")
+def hello():
+    """Somebody is saying hi, how lovely!"""
+    display.show(Image.HEART)
+    sleep(400)
+```
+
+When the bird detects a hello message from the airwaves, it will run this
+function and show a heart image in the micro:bit display for 400 milliseconds.
+
+<p class="exercise">
+Exercise: <br>
+Birds can be very chatty, so your neighbours might be saying hello quite often.
+<br>
+Can you think of a way to alternate between showing `Image.HEART` and
+`Image.HEART_SMALL`? That way it'll look like a beating heart!
+</p>
+
+[Solution](https://github.com/microbit-carlos/microbit-bird-activity/tree/main/solutions/1-hello-beating.md)
+
+When you code is ready, click the purple "Send to micro:bit" button.
+If WebUSB is not available in your browser, you can click the "Save" button
+to download a hex file, and then copy that file into the `MICROBIT` USB drive.
+
+### What was that! A cat?
+
+There are several events the bird can react to, and seeing a cat is quite important for bird survival.
+
+<p class="exercise">
+Exercise: <br>
+Let's do the following:<br>
+- Copy and paste the "hello" example <br>
+- Replace the function name to `cat()` <br>a
+- Change the decorator argument from "hello" to "cat".<br>
+<br>
+What do you think the bird should do in this case?
+<br>
+Some examples of thing you can do can be found in the
+<a href="https://microbit-carlos.github.io/microbit-bird-activity/micropython/#quick-things-to-do-with-the-microbit" target="_blank">MicroPython page</a>.
+</p>
+
+We also want to warn other birds around us,
```

Comment:
```suggestion
Assuming we're the kind of bird that likes our local fellow birds, we also want to warn other birds around us,
```

Reply:
Added in 21ad6bda79cb278392ffcc3482dd4da27274a3cb